packages/core/src/jsx.js

```diff
@@ -15,20 +15,15 @@ export const jsx: typeof React.createElement = function(
     return React.createElement.apply(undefined, args)
   }
 
-  const emotionProps = createEmotionProps(type, props)
+  let argsLength = args.length
+  let createElementArgArray = new Array(argsLength)
+  createElementArgArray[0] = Emotion
+  createElementArgArray[1] = createEmotionProps(type, props)
 
-  // https://github.com/facebook/react/blob/fd61f7ea53989a59bc427603798bb111c852816a/packages/react/src/ReactElement.js#L386-L400
```

Comment:
I'm kinda not sure why I have changed this logic to replicate what React is doing under the hood. I've implemented support for those new runtimes and refactor to the classic `jsx` a few months back, so 🤷‍♂️